exchanges/gateio/gateio_convert.go

```diff
@@ -451,15 +451,15 @@ func (a *Orderbook) UnmarshalJSON(data []byte) error {
 		*Alias
 		Current float64    `json:"current"`
 		Update  float64    `json:"update"`
-		Bids    []askorbid `json:"asks"`
-		Asks    []askorbid `json:"bids"`
+		Asks    []askorbid `json:"asks"`
+		Bids    []askorbid `json:"bids"`
 	}{}
 	err := json.Unmarshal(data, &chil)
 	if err != nil {
 		return err
 	}
-	a.Current = time.Unix(int64(chil.Current), 0)
-	a.Update = time.Unix(int64(chil.Update), 0)
+	a.Current = time.UnixMilli(int64(chil.Current * 1000))
```

Comment:
This is still failing from [here](https://github.com/thrasher-corp/gocryptotrader/pull/1058#discussion_r1022249653)

```
Current:54849-11-03 03:00:21 +1100 AEDT Update:54849-11-03 03:00:13 +1100 AEDT
```